github-actions/setup-bazel-remote-exec/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Setup Bazel remote execution'
+description: 'Sets up Bazel remote execution.'
+author: 'Angular'
+inputs:
+  bazelrc:
```

Comment:
Can we have a default here since its usually the same place?

Reply:
Not all repositories use the same pattern where `--config=remote` should live, so it's not a default (similar in the orb)

Reply:
👍